tests/unit/aiplatform/test_pipeline_jobs.py

```diff
@@ -1467,6 +1473,7 @@ def test_clone_pipeline_job_with_all_args(
             template_path=_TEST_TEMPLATE_PATH,
             job_id=_TEST_PIPELINE_JOB_ID,
             parameter_values=_TEST_PIPELINE_PARAMETER_VALUES,
+            input_artifacts=_TEST_PIPELINE_INPUT_ARTIFACTS,
```

Comment:
Could you please remove this line? The purpose of this test is to check if clone method can update `input_artifacts` while cloning an existing job. I think it makes more sense to create a job without setting `input_artifacts`, clone it with artifacts being set, and check if the cloned job has the setting.

Reply:
Done.